<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormBD</class>
 <widget class="QWidget" name="FormBD">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>684</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>684</width>
    <height>301</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Форма добавления/ удаления</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources/images.qrc">
    <normaloff>:/form.jpeg</normaloff>:/form.jpeg</iconset>
  </property>
  <widget class="QComboBox" name="mode">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>add</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>delete</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="line_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelName">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_adress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelAdress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>adress:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_phone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>351</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelPhone">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>phone:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonPushRequest">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Push request</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="authorizationGroup">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Authorization</string>
   </property>
   <widget class="QLabel" name="labelPassword">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_login">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLogin">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>login:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonLogIn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>log in</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resources/images.qrc"/>
 </resources>
 <connections/>
</ui>
